support-frontend/assets/pages/paper-subscription-landing/components/content/info.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="19" height="19" viewBox="0 0 19 19" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
just wanted to check this has been through the svgo tool to make sure it's compacted as much as possible

Reply:
No it hasn't, and I didn't know about that. Tell me more!

Reply:
I think you did it before I saw this, I mean this https://jakearchibald.github.io/svgomg/

Reply:
this could be a place for tooling for the quality okr, atomatically check in a PR to make sure images are optimised